pageserver/src/tenant/throttle.rs

```diff
@@ -58,6 +58,11 @@ pub struct Stats {
     pub sum_throttled_usecs: u64,
 }
 
+pub enum ThrottleResult {
+    NotThrottled { start: Instant },
```

Comment:
nit: `start` is a confusing name. What started at said instant? I'd go for `at` or smth like that.

Reply:
decided to not re-run